src/ooxml/java/org/apache/poi/xssf/streaming/SXSSFRow.java

```diff
@@ -132,6 +132,24 @@ public SXSSFCell createCell(int column)
      * @return Cell a high level representation of the created cell.
      * @throws IllegalArgumentException if columnIndex < 0 or greate than a maximum number of supported columns
      * (255 for *.xls, 1048576 for *.xlsx)
+     * @deprecated POI 3.15 beta 3. Use {@link #createCell(int, CellType)} instead.
+     */
+    @Override
+    public SXSSFCell createCell(int column, int type)
```

Comment:
As per previous comment, I don't understand why this is being added back.

Reply:
also removed again